src/components/inspector/panels/Panels.tsx

```diff
@@ -96,6 +97,10 @@ const Panels: React.FC<{ component: IComponent; isRoot: boolean }> = ({
       {type === 'Radio' && <RadioPanel />}
       {type === 'RadioGroup' && <RadioGroupPanel />}
       {type === 'Select' && <SelectPanel />}
+      {type === 'Slider' && <SliderPanel />}
+      {type === 'SliderTrack' && <ChildrenControl />}
+      {type === 'SliderFilledTrack' && <ChildrenControl />}
+      {type === 'SliderThumb' && <ChildrenControl />}
```

Comment:
I think it will be better to do it like this
```
 {(type === 'SliderTrack' || type === 'SliderFilledTrack'  || type === 'SliderThumb') && <ChildrenControl />}
```